### PR TITLE
fix: kvcountdowtimer component updated to show the remaining hours co…

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- Showing more than 24 hrs in our KvCountdownTimer component

Screenshot 2023-10-17 at 16 55 58

### CODE REVIEW

File: @kiva/kv-components/vue/stories/KvCountdownTimer.stories.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ const story = (args) => {
 
 export const Seconds = story({ msLeft: 1000 * 10 });
 
-// One second less than 24 hours (since we don't show days in timer)
-export const Tomorrow = story({ msLeft: 1000 * 60 * 60 * 24 - 1000 });
+export const Tomorrow = story({ msLeft: 1000 * 60 * 60 * 24 });
 
 export const Expired = story({ msLeft: 0 });
+
+export const Overmorrow = story({ msLeft: 1000 * 60 * 60 * 48 });
```
Comment (dyersituations) [inline @kiva/kv-components/vue/stories/KvCountdownTimer.stories.js:30]: Nice old school English!

File: @kiva/kv-components/vue/KvCountdownTimer.vue [hunk 2]
```diff
@@ -26,6 +27,10 @@ export default {
 		const interval = ref(null);
 		const timeLeft = ref(null);
 
+		const remainingHours = computed(() => {
+			return Math.floor(timeLeft.value.asHours());
```
Comment (dyersituations) [inline @kiva/kv-components/vue/KvCountdownTimer.vue:31]: How does this differ from `.hours()`?

Comment (roger-in-kiva) [reply @kiva/kv-components/vue/KvCountdownTimer.vue:31]: Here the answer from documentation

```
As with the other getters for durations, moment.duration().hours() gets the hours (0 - 23).

moment.duration().asHours() gets the length of the duration in hours.
```

Comment (dyersituations) [reply @kiva/kv-components/vue/KvCountdownTimer.vue:31]: Makes sense, thanks! Hopefully we can easily replace this with `date-fns` methods later...

**Review by dyersituations — APPROVED** [review]

File: @kiva/kv-components/vue/KvCountdownTimer.vue [hunk 3]
```diff
@@ -10,6 +10,7 @@ import {
 	toRefs,
 	onBeforeUnmount,
 	onMounted,
+	computed,
 } from 'vue-demi';
 import moment from 'moment';
```
Comment (mcstover) [inline @kiva/kv-components/vue/KvCountdownTimer.vue:15]: We had discussed replacing this with date-fns (used in both ui + cps, supports granular imports) so we aren't importing the entire moment library. Is that some work you could ticket out @roger-in-kiva ?

Comment (roger-in-kiva) [reply @kiva/kv-components/vue/KvCountdownTimer.vue:15]: Nice to know @mcstover !
I added a ticket for the next sprint https://kiva.atlassian.net/browse/CORE-1596

Comment (mcstover) [reply @kiva/kv-components/vue/KvCountdownTimer.vue:15]: Thank you!

**Pull request merged**